tests/annotated_binary/annotated_binary_old.afb

```diff
@@ -0,0 +1,293 @@
+  +0x0060 | 0C 00 00 00             | uint32_t     | 0x0000000C (12)                    | struct field `AnnotatedBinary.Building.windows` (Int)
+  +0x0064 | 0A 00 00 00             | uint32_t     | 0x0000000A (10)                    | array field `AnnotatedBinary.Dimension.values[0]` (Int)
+  +0x0068 | 0C 00 00 00             | uint32_t     | 0x0000000C (12)                    | array field `AnnotatedBinary.Dimension.values[1]` (Int)
+  +0x006C | 14 00 00 00             | uint32_t     | 0x00000014 (20)                    | array field `AnnotatedBinary.Dimension.values[2]` (Int)
+  +0x0070 | 01                      | uint8_t      | 0x01 (1)                           | struct field `AnnotatedBinary.Tolerance.width` (UByte)
+  +0x0071 | 02                      | uint8_t      | 0x02 (2)                           | struct field `AnnotatedBinary.Tolerance.width` (UByte)
+  +0x0072 | 03                      | uint8_t      | 0x03 (3)                           | struct field `AnnotatedBinary.Tolerance.width` (UByte)
+  +0x0073 | 00                      | uint8_t[1]   | .                                  | padding
+  +0x0074 | C8 01 00 00             | UOffset32    | 0x000001C8 (456) Loc: +0x023C      | offset to field `name`
+  +0x0078 | 5C 01 00 00             | UOffset32    | 0x0000015C (348) Loc: +0x01D4      | offset to field `bars`
+  +0x007C | 50 01 00 00             | UOffset32    | 0x00000150 (336) Loc: +0x01CC      | offset to field `bar_baz` (union of type `Baz`)
+  +0x0080 | 34 01 00 00             | UOffset32    | 0x00000134 (308) Loc: +0x01B4      | offset to field `accounts`
+  +0x0084 | 24 01 00 00             | UOffset32    | 0x00000124 (292) Loc: +0x01A8      | offset to field `bob`
+  +0x0088 | 14 01 00 00             | UOffset32    | 0x00000114 (276) Loc: +0x019C      | offset to field `alice`
+  +0x008C | 0D 00 00 00             | uint32_t     | 0x0000000D (13)                    | table field `just_i32` (Int)
+  +0x0090 | DC 00 00 00             | UOffset32    | 0x000000DC (220) Loc: +0x016C      | offset to field `names`
+  +0x0094 | A0 00 00 00             | UOffset32    | 0x000000A0 (160) Loc: +0x0134      | offset to field `points_of_interest`
+  +0x0098 | 94 00 00 00             | UOffset32    | 0x00000094 (148) Loc: +0x012C      | offset to field `foobars_type`
+  +0x009C | 38 00 00 00             | UOffset32    | 0x00000038 (56) Loc: +0x00D4       | offset to field `foobars`
+  +0x00A0 | 33 00 00 00             | UOffset32    | 0x00000033 (51) Loc: +0x00D3       | offset to field `measurement` (union of type `Tolerance`)
+  +0x00A4 | 1C 00 00 00             | UOffset32    | 0x0000001C (28) Loc: +0x00C0       | offset to field `anything` (union of type `Bar`)
+  +0x00A8 | 04 00 00 00             | ?uint8_t[4]  | ....                               | Unknown field <possibly an offset? Check Loc: +0x000000AC>
```

Comment:
@aardappel this is what I put in the latest PR

Reply:
I just put it as a unknown field. And since the size == sizeof(uoffset32) i provided a hint to check the location of where it would point to.